server/views/components/wip/renderField/_conditionalHtml.njk

```diff
@@ -21,7 +21,7 @@ USAGE:
         text: "Date"
       },
       value: values[field.name + '-' + item.id + '-date'],
-      leadingZeros: "true"
+      leadingZeros: true
```

Comment:
Does this work now? I don't think they fixed it yet…?

https://github.com/ministryofjustice/moj-frontend/issues/780

Reply:
Main reason I changed to a boolean is because when it was a string the dake picker was not rendering correctly.

Oddly, when the property is a string it throws a `Uncaught TypeError: this.config.leadingZeros.toLowerCase is not a function` (which is odd because surely a string would have the `toLowerCase()` method):

```JavaScript
Datepicker.prototype.setLeadingZeros = function () {
  if (typeof this.config.leadingZeros !== "boolean") {
    if (this.config.leadingZeros.toLowerCase() === "true") {
      this.config.leadingZeros = true;
    }
    if (this.config.leadingZeros.toLowerCase() === "false") {
      this.config.leadingZeros = false;
    }
  }
};
```

`leadingZeros: true` doesn't seem to make much difference at the moment AFAICT but at least the date picker works.